include/SDL3/SDL_gpu.h

```diff
@@ -189,14 +189,14 @@ typedef Uint32 SDL_GpuTransferBufferMapFlags;
 typedef enum SDL_GpuShaderStage
 {
     SDL_GPU_SHADERSTAGE_VERTEX,
-    SDL_GPU_SHADERSTAGE_FRAGMENT,
-    SDL_GPU_SHADERSTAGE_COMPUTE
+    SDL_GPU_SHADERSTAGE_FRAGMENT
 } SDL_GpuShaderStage;
```

Comment:
We may need to specify _graphics_ shader stage, so that it's clear that this is separate from compute which is still supported